src/python/foglamp/translators/omf_translator.py

```diff
@@ -896,88 +875,129 @@ def omf_object_creation():
 
                         message = "### sensor information ##################################################"
-                        debug_msg_write("INFO", "{0}".format(message))
+                        _logger.debug("{0}".format(message))
 
                         # Identification of the object/sensor
-                        _sensor_id = db_row.asset_code
-                        _measurement_id = "measurement_" + _sensor_id
+                        sensor_id = db_row.asset_code
+                        measurement_id = "measurement_" + sensor_id
 
+                        tmp_type = ""
                         try:
-                            tmp_type = _sensor_name_type[_sensor_id]
+                            # Evaluates if it is a known types
+                            tmp_type = _sensor_name_type[sensor_id]
 
                         except Exception as e:
-                            message = _message_list["e000012"].format(e)
+                            message = _message_list["e000012"].format(tmp_type, e)
 
-                            _log.error(message)
-                            debug_msg_write("WARNING", "{0}".format(message))
+                            _logger.warning(message)
                         else:
-                            _type_sensor_id = "type_sensor_id_" + _type_id + "_" + tmp_type
-                            _type_measurement_id = "type_measurement_" + _type_id + "_" + tmp_type
+                            _logger.debug("db row |{0}| |{1}| |{2}| ".format(db_row.id,
+                                                                             db_row.user_ts,
+                                                                             db_row.reading))
+
+                            try:
+                                # Loads data into the PI Server using OMF
+                                values = create_data_values_stream_message(measurement_id, db_row)
+                                send_omf_message_to_end_point("Data", values)
+
+                                # at least one row sent to OMF
+                                info_handled = True
+
+                                # Updates statistics
+                                _num_sent += 1
 
-                            debug_msg_write("INFO", "OMF_object_creation ")
-                            omf_object_creation()
+                                # Updates latest position sent
+                                new_position = db_row.id
 
-                            debug_msg_write("INFO", "db row |{0}| |{1}| |{2}| ".format(db_row.id,
-                                                                                       db_row.user_ts,
-                                                                                       db_row.reading))
+                            except Exception as e:
+                                # Updates statistics
+                                _num_unsent += 1
```

Comment:
this _num_unsent is for internal reason it is not related (requested) for the statistics.